code/client/clrcore/External/Blip.cs

```diff
@@ -498,7 +639,20 @@ private void _Delete()
 				Handle = handle;
 			}
 		}
+#elif IS_FXSERVER
+		[SecuritySafeCritical]
+		private void _Delete()
+		{
+			int handle = Handle;
+			unsafe
```

Comment:
Unrelatedly, I'm not sure if this needs an `unsafe` block as such.

Also, server blip natives do need some extra 'love' but I guess this is a simple file to check if the `#if` stuff made sense.

Reply:
yeah it was a draft to see if the #if stuff would make sense in this context